tests/test_addon_detail.py

```diff
@@ -495,3 +495,81 @@ def test_click_addon_in_more_addons_by_author(selenium, base_url, variables):
     addon.same_author_addons.addons_by_author_results_items[0].click()
     addon.wait_for_page_to_load()
     assert result_name in addon.name
+
+
+@pytest.mark.nondestructive
+def test_addon_description(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert 'About this extension' in addon.description.addon_description_header
+    assert addon.description.addon_description_text.is_displayed()
+
+
+@pytest.mark.nondestructive
+def test_addon_ratings_card(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get('{}/addon/{}'.format(base_url, extension))
```

Comment:
This looks like it could be in f string format

Reply:
yeah, I don't know where I've lost the 'f'. I'll fix that.